docs/modules/casper.rst

```diff
@@ -410,7 +437,7 @@ Also have a look at `forward()`_.
 ``base64encode()``
 -------------------------------------------------------------------------------
 
-**Signature:** ``base64encode(String url [, String method, Object data])``
+**Signature:** ``base64encode(String url [, String method, Object data, Boolean async])``
```

Comment:
This would give the impression that the third parameter is required when the second is given.
Perhaps `(String url [, String method [, Object data, Boolean async ] ] )`